scss/detail.scss

```diff
@@ -1,6 +1,11 @@
 .book-container {
+  background-color: $white;
+  -webkit-box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
   font-family: 'Roboto', sans-serif;
   padding: 15px;
+  height: 416px;
+  width: 900px;
```

Comment:
According to our style guide, for responsive reasons, if an element's width is higher than `250px` is better to set 

```css
width: 100%;
max-width: 900px;
```
 
That way, it will stretch to the entire screen when it gets smaller than 900px